Libro.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require_once 'Autor.php';
+require_once 'IPublicable.php';
+
+class Libro implements IPublicable {
+  
+    private string $titulo;
+    private int $anioPublicacion;
+    private Autor $autor; 
+
+    public function __construct(string $titulo, int $anioPublicacion, Autor $autor) {
+        $this->titulo = $titulo;
+        $this->anioPublicacion = $anioPublicacion;
+        $this->autor = $autor;
+    }
+
+    
+    public function getTitulo(): string {
+        return $this->titulo;
+    }
```

Comment:
```suggestion
    }

```
Agregar espacio en blanco